main.py

```diff
@@ -1,33 +1,59 @@
-
 from plothandler import *                           #Import the server that plots the result in browser
 from shortest_path_random_walk.shortest_path_random_walk_new import *         #Import everything from shortest_path_random_walk
```

Comment:
If you want this code to "survive" future updates, you should add it to its own file (maybe in the ppo2_shortpath folder?), import it like the "shortest_path_random_walk" above and instantiate it. However, if the code here in main is not that important, you can leave it here, but it will almost centainly be overwritten later.

Reply:
Yes I'll do that. The code is not very important, was just so we could run some models